plugins/modules/auth_source_ldap.py

```diff
@@ -215,14 +215,19 @@ def main():
             groups_base=dict(),
             server_type=dict(choices=["free_ipa", "active_directory", "posix"]),
             ldap_filter=dict(),
-            use_netgroups=dict(type='bool'),
+            ldap_group_membership=dict(choices=["posix", "nis_netgroups", "rfc4519"]),
         ),
         required_if=[['onthefly_register', True, ['attr_login', 'attr_firstname', 'attr_lastname', 'attr_mail']]],
     )
 
     # additional parameter checks
-    if 'use_netgroups' in module.foreman_params and module.foreman_params['server_type'] == 'active_directory':
-        module.fail_json(msg='use_netgroups cannot be used when server_type=active_directory')
+    server_type = module.foreman_params['server_type']
```

Comment:
server_type is not a required param, so this can be absent.
```suggestion
    server_type = module.foreman_params.get('server_type', 'posix')
```

Reply:
Oh, right, fixed